go/vt/tabletserver/query_splitter.go

```diff
@@ -97,7 +98,7 @@ func (qs *QuerySplitter) validateQuery() error {
 				}
 			}
 		}
```

Comment:
This file is not being used anymore and I forgot to remove it; sorry for causing you some extra work. While you're in here, can you please delete it and the associated test.